harness/determined/_experiment_config.py

```diff
@@ -8,9 +8,6 @@ def debug_enabled(self) -> bool:
     def scheduling_unit(self) -> int:
         return int(self.get("scheduling_unit", 100))
 
-    def native_enabled(self) -> bool:
-        return "internal" in self and self["internal"] is not None and "native" in self["internal"]
-
     def native_parallel_enabled(self) -> bool:
```

Comment:
completely unrelated, but what even is this?

Reply:
more dead code, I'm pretty sure.  Actually, now that I look around it's totally unused, so I'll just remove it.